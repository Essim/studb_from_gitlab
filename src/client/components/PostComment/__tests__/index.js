import React from 'react';
import {PostComment} from '../';
import { shallowToJson } from 'enzyme-to-json';

const mockUser={
        pseudo:'mockPseudo',
        cuid:'mockCuid',
    },mockDocument="mockDocument",
    mockConnected=true,
    mockCommentDocument = jest.fn(),
    mockCommenting={
        loading:false,
    };

describe('PostComment', () => {
    it('should render correctly', () => {
        const output = shallow(<PostComment user={mockUser} connected={mockConnected} document={mockDocument} commentDocument={mockCommentDocument} commenting={mockCommenting}/>);
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});
