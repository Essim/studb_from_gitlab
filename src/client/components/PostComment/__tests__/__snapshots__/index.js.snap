// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostComment should render correctly 1`] = `
<form
  action="#"
  method="post"
  onSubmit={[Function]}
>
  <Gravatar
    className="img-responsive img-circle img-sm"
    default="identicon"
    email="mockPseudo"
    protocol="//"
    rating="g"
    size={50}
  />
  <div
    className="img-push"
  >
    <Input
      className="form-control input-sm"
      onChange={[Function]}
      onFocus={[Function]}
      placeholder="Lache un commentaire !"
      tag="p"
      type="text"
      value=""
    />
  </div>
</form>
`;
